src/vm/methodtable.cpp

```diff
@@ -9354,11 +9353,8 @@ void MethodTable::SetSlot(UINT32 slotNumber, PCODE slotCode)
         if (fSharedVtableChunk)
         {
             MethodDesc* pMD = GetMethodDescForSlotAddress(slotCode);
-#ifndef FEATURE_INTERPRETER
```

Comment:
Just curious, is there a reason to remove FEATURE_INTERPRETER in a number of places?